### ISSUE TITLE
[Accessibility] In High Contrast Mode focus is not visible on “clear value” and “calendar" button.

### ISSUE THREAD

**Issue by asthanarht** [kekeh/mydatepicker#623]
Environment Details: 
Application Name: Microsoft Collaborate
Application Url: https://partner.microsoft-int.com/en-us/dashboard/collaborate/
Windows Version: Windows10
Browser Version: Microsoft Edge 44.17763.1.0

 Repro Steps:​ 
1. Launch page having mydatepicker control
2. Navigate to Next button
3. Navigate by using keyboard.
4.Verify that keyboard focus is visible in high contrast mode.

Actual:
In High Contrast Mode focus is not visible on “clear value” and “calendar" button.

Expected:
Focus should be visible in high contrast mode for all the controls present on the page.

User Impact:​
Low vision/color blind user will find it difficult to identify where the current focus is when navigated to above specified .
![image](https://user-images.githubusercontent.com/4268434/66688031-bd9d3b80-ec39-11e9-9f34-d1a5efb70bc4.png)
